open-metadata-implementation/access-services/data-engine/data-engine-server/src/main/java/org/odpi/openmetadata/accessservices/dataengine/server/handlers/DataEngineRegistrationHandler.java

```diff
@@ -149,4 +163,65 @@ ExternalDataEnginePropertiesBuilder getExternalDataEnginePropertiesBuilder(Softw
                 softwareServerCapability.getEngineVersion(), softwareServerCapability.getPatchLevel(), softwareServerCapability.getSource(),
                 softwareServerCapability.getAdditionalProperties(), repositoryHelper, serviceName, serverName);
     }
+
+    public void createDataEngineClassification(String userId, ProcessingState processingState, String externalSourceName) throws
+            InvalidParameterException,
+            UserNotAuthorizedException,
+            PropertyServerException, EntityNotKnownException {
+        final String methodName = "createDataEngineClassification";
+
+        invalidParameterHandler.validateUserId(userId, methodName);
+
+        String externalEngineGUID = this.getExternalDataEngine(userId, externalSourceName);
+        if (externalEngineGUID == null) {
+            ExceptionMessageDefinition messageDefinition = DataEngineErrorCode.SOFTWARE_SERVER_CAPABILITY_NOT_FOUND.getMessageDefinition(externalSourceName);
+            throw new EntityNotKnownException(messageDefinition, this.getClass().getName(),
```

Comment:
Access service should not throw repository service exception ....

Reply:
fixed, now throwing InvalidParameterException from org.odpi.openmetadata.frameworks.connectors.ffdc